### ISSUE TITLE
}bedrock.hier.consol.flat.create - not working because of missing parameter

### ISSUE THREAD

**Issue by scrumthing** [cubewise-code/bedrock#228]
I start the above mentioned process with the following parameters. 

```
ExecuteProcess('}bedrock.hier.consol.flat.create',
  'pLogOutput', pLogOutput,
  'pDim',pDim,
  'pHier',pHier,
  'pSrcConsol','',
  'pTgtConsol',pRootEle,
  'pWeight',1);
```

But instead of a succes, I get the following error: **Variable "pConsol" not found**

The parameter pConsol does not exist in the process defintion. It is only referenced in logging parts. 

EDIT:
Wrong snippet... Sorry for that...

![Bedrock_Error_hierarchie_create](https://user-images.githubusercontent.com/8442014/101790563-e6051c00-3b02-11eb-8926-137c822cc03e.PNG)

**Comment by lotsaram** [comment]
Hi @scrumthing 
I can't replicate this and find no reference to "pConsol" anywhere in the code for the process. 
What version of the process are you using?
Are you sure this message isn't coming from the calling process somewhere?

**Comment by scrumthing** [comment]
Sorry, copied the wrong process. Edited the request. 
**}bedrock.hier.consol.flat.create**
[Line 127](https://github.com/cubewise-code/bedrock/blob/master/main/%7Dbedrock.hier.consol.flat.create.pro)

**Comment by scrumthing** [comment]
cannot reopen @lotsaram. Should I create a new one?

**Comment by lotsaram** [comment]
Fixed in #248
